libs/utils/action.js

```diff
@@ -8,3 +8,4 @@ export function debounce(callback, time = 300) {
     timer = setTimeout(() => callback(...args), time);
   };
 }
+export default { debounce };
```

Comment:
Nit: we always leave a space between function declarations and/or exports, so there should be a new line between L10 and L11